pkg/agent/apiserver/handlers/networkpolicy/handler.go

```diff
@@ -80,12 +81,16 @@ func newFilterFromURLQuery(query url.Values) (*querier.NetworkPolicyQueryFilter,
 	}
 
 	source := query.Get("source")
-
 	name := query.Get("name")
 	if name != "" && (source != "" || namespace != "" || pod != "" || strSourceType != "") {
 		return nil, fmt.Errorf("with a name, none of the other fields can be set")
 	}
 
+	sortBy := query.Get("sort-by")
+	if sortBy != "" && sortBy != sortByEffectivePriority {
+		return nil, fmt.Errorf("unsupported value for sort-by option. Current supported value is %s", sortByEffectivePriority)
+	}
```

Comment:
@Dyanngg I am confused as to why the check is done on the server-side, and only for the agent?

Reply:
You are totally right. Reverted that and added that check on the client-side